### PR TITLE
Remove builtins

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

**Comment by LK4D4** [comment]
COOL

File: integration-cli/docker_cli_run_test.go [hunk 1]
```diff
@@ -3363,7 +3363,7 @@ func TestRunRestartMaxRetries(t *testing.T) {
 		t.Fatal(string(out), err)
 	}
 	id := strings.TrimSpace(string(out))
-	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 5); err != nil {
+	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 10); err != nil {
```
Comment (duglin) [inline integration-cli/docker_cli_run_test.go]: why the change from 5 to 10?

Comment (runcom) [inline integration-cli/docker_cli_run_test.go]: cause I was running test and I think because laptop was under heavy load (compiling kernel) and this tests timed out, I don't know if this was the problem, test runs ok now, I can change it back

File: integration/runtime_test.go [hunk 2]
```diff
@@ -157,9 +158,9 @@ func spawnGlobalDaemon() {
 			Scheme: testDaemonProto,
 			Host:   testDaemonAddr,
 		}
-		job := eng.Job("serveapi", listenURL.String())
-		job.SetenvBool("Logging", true)
-		if err := job.Run(); err != nil {
+
+		serverConfig := &apiserver.ServerConfig{Logging: true}
```
Comment (LK4D4) [inline integration/runtime_test.go]: I dunno why it was `true`, but should be really `false`. And below too. I hope we will kill `integration` suite soon :/

Comment (runcom) [inline integration/runtime_test.go]: Make sense

**Comment by tiborvass** [comment]
I :heart: @runcom

File: integration-cli/docker_cli_run_test.go (hunk 1 above)
Comment (LK4D4) [inline integration-cli/docker_cli_run_test.go]: I think it is okay. It won't hurt fast machines, but your laptop should pass tests too.

Comment (runcom) [inline integration-cli/docker_cli_run_test.go]: Yeah it currently runs ok even with 5 seconds, but was under heavy load (make -j14) and I suppose it was timing out for this

**Comment by runcom** [comment]
@tiborvass :)

**Comment by duglin** [comment]
LGTM
very exciting!

**Comment by LK4D4** [comment]
LGTM
I think my comment changes nothing.

**Comment by runcom** [comment]
cool just merged

File: integration/runtime_test.go (hunk 2 above)
Comment (runcom) [inline integration/runtime_test.go]: @LK4D4 in the meantime I can create a beginner issue to change it to false

Comment (LK4D4) [inline integration/runtime_test.go]: Nah, integration calls `createRouter` directly anyway. I think it is probably easier just remove this field and use `true` and `false` values where it is needed for `createRouter`.

Comment (runcom) [inline integration/runtime_test.go]: I'll handle that!
